### PR TITLE
Ga prototyping docs

### PR DESCRIPTION
**Pull request by ig-georgeA**
Updating video tutorial links and topic titles.

### CODE REVIEW

File: en/video-tutorials.md [hunk 1]
```diff
@@ -45,7 +45,29 @@ _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Nati
...
+            <iframe width="400" height="225" src="https://www.youtube.com/watch?v=UK0vZTKCt8I&list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=3" frameborder="0" allowfullscreen></iframe>
+            <p> Take part in a user test</p>
+            <br>
+        </div>
+        </div>
+          <div class="feature__image">
+            <iframe width="400" height="225" src="https://www.youtube.com/watch?v=hdVHP01_c1I&list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=4" frameborder="0" allowfullscreen></iframe>
+            <p> Review user-testing results</p>
+            <br>
+        </div>
```
Comment (StefanIvanov) [inline en/video-tutorials.md:68]: These 4 videos don't load
Screenshot 2020-07-20 at 19 36 49

Comment (ig-georgeA) [reply en/video-tutorials.md:68]: Dang it... checking.

Comment (pmoleri) [reply en/video-tutorials.md]: Addressed.
Changed `/watch?v=<id>&list=...`
by            `/embed/<id>?list=...`

File: en/prototyping/review-user-testing-results.md [hunk 2]
```diff
@@ -98,19 +98,19 @@ And to download the video, right click on the video use the "save video as..." o
...
-* [Take part in a usability study](https://www.youtube.com/watch?v=dfrxhBW5NFQ?rel=0&autoplay=1)
-* [View usability study results](https://www.youtube.com/watch?v=00P6DwMbYMM?rel=0&autoplay=1)
+* [Set up a user-test](https://www.youtube.com/watch?v=_jCDjtxTLCo&list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=2)
+* [Take part in a user-test](https://www.youtube.com/watch?v=UK0vZTKCt8I&list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=3)
+* [Review user-testing results](https://www.youtube.com/watch?v=hdVHP01_c1I&list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=4)
 
 
 [1]: https://www.youtube.com/watch?v=vilyDL4fDT0
 [2]: https://cloud.indigo.design/api/shares/qv6uzwx9jwu3/files/project/Money%20App.zip
 [3]: https://cloud.indigo.design
```
Comment (StefanIvanov) [inline en/prototyping/review-user-testing-results.md:108]:
```suggestion

```

Seem not to be used in the topic

File: en/prototyping/workspaces.md [hunk 3]
```diff
@@ -84,4 +84,4 @@ There's a new participant in a workspace usability study | Subscribe to notifica
 [a-4]: #managing-group-workspaces
 [a-5]: #configuring-notifications
 
-[topic-1]: setting-up-a-usability-study.md
+[topic-1]: set-up-a-user-test.md
```
Comment (StefanIvanov) [inline en/prototyping/workspaces.md:87]: Change  [Setting Up a Usability Study][topic-1] to * [Setting Up a User Test][topic-1]

and remove the a-1 to a-5 references as they are not used in the topic above

File: en/sync-themes-plugin.md [hunk 4]
```diff
@@ -111,8 +111,8 @@ Instead of publishing a new prototype, you may want to update an existing one by
 
 Once the prototype is successfully published to the cloud, you will be handed the link to it plus options to:
 
-- view it in the workspace it belongs to, which will opent it up in a browser window
-- create usability study, which will initialize a [remote usability study](prototyping/setting-up-a-usability-study.md) and open it in a browser window for further configuration
+- view it in the workspace it belongs to, which will open it up in a browser window
+- create usability study, which will initialize a remote [usability study](prototyping/set-up-a-user-test.md) and open it in a browser window for further configuration
```
Comment (StefanIvanov) [inline en/sync-themes-plugin.md:115]:
```suggestion
- create usability study, which will initialize a [user test](prototyping/set-up-a-user-test.md) and open it in a browser window for further configuration
```

**Comment by StefanIvanov** [comment]
@pmoleri @ig-georgeA these should be the final changes we need to make

File: en/sync-themes-plugin.md (hunk 4 above)
Comment (pmoleri) [reply en/sync-themes-plugin.md:115]: Usability Study was kept there because if was the actual button displays.
@ig-georgeA could you confirm?

Comment (ig-georgeA) [reply en/sync-themes-plugin.md:115]: @pmoleri sorry. It should say ..."Create a usability test"

Comment (StefanIvanov) [reply en/sync-themes-plugin.md:115]: @ig-georgeA 
like this?
create a usability test, which will initialize a remote [user test]
see above the suggested change too and accept it if it is OK

File: en/prototyping/workspaces.md (hunk 3 above)
Comment (StefanIvanov) [reply en/prototyping/workspaces.md:87]: @ig-georgeA any comments here?

Comment (ig-georgeA) [reply en/prototyping/workspaces.md:87]: @StefanIvanov thanks for waiting. Getting familiar with the github workflows. :)

File: en/video-tutorials.md [hunk 5]
```diff
@@ -45,7 +45,29 @@ _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Nati
...
+        </div>
+          <div class="feature__image">
+            <iframe width="400" height="225" src="https://www.youtube.com/embed/hdVHP01_c1I?list=PLZ4rRHIJepBs9bwRs1LZLV0ZVzlFKqwCw&index=4" frameborder="0" allowfullscreen></iframe>
+            <p> Review user-testing results</p>
+            <br>
+        </div>
     </div>
+     
 </section>
```
Comment (ig-georgeA) [inline en/video-tutorials.md:72]: @pmoleri were u able to add the correct embed links? Is that what I am seeing here?

File: en/prototyping/workspaces.md (hunk 3 above)
Comment (StefanIvanov) [reply en/prototyping/workspaces.md:87]: I think this is not resolved yet:
Change [Setting Up a Usability Study][topic-1] to * [Setting Up a User Test][topic-1] and remove the a-1 to a-5 references as they are not used in the topic above

File: en/prototyping/workspaces.md [hunk 6]
```diff
@@ -77,11 +77,4 @@ There's a new participant in a workspace usability study | Subscribe to notifica
 
 * [Setting Up a Usability Study][topic-1]
```
Comment (StefanIvanov) [inline en/prototyping/workspaces.md:78]:
```suggestion
* [Setting Up a Usability Test][topic-1]
```

**Pull request merged**